python/hsfs/core/feature_monitoring_config_engine.py

```diff
@@ -60,19 +63,24 @@ def __init__(
             assert feature_view_id is not None
             assert feature_view_name is not None
             assert feature_view_version is not None
+            entity_type = "featuregroups"
+        else:
+            entity_type = "featureview"
 
         self._feature_monitoring_config_api = FeatureMonitoringConfigApi(
             feature_store_id=feature_store_id,
             feature_group_id=feature_group_id,
             feature_view_name=feature_view_name,
             feature_view_version=feature_view_version,
         )
+        self._statistics_api = StatisticsApi(feature_store_id, entity_type)
```

Comment:
Do we want the engine to depend on the statistics api directly rather than only on the engine? Is it temporary maybe?

Reply:
Yes, it was temporary. But I will fixed it now to avoid confusion